muffnn/mlp/tests/test_mlp_regressor.py

```diff
@@ -139,3 +140,24 @@ def test_embedding_specific_layer():
     clf.fit(X, y)
 
     assert clf.transform(X).shape[1] == 8
+
+
+def test_prediction_gradient():
+    """Test computation of prediction gradients."""
+    mlp = MLPRegressor(n_epochs=100, random_state=42, hidden_units=(5,))
+    X, y = make_regression(
+        n_samples=1000, n_features=10, n_informative=1, shuffle=False)
+    mlp.fit(X, y)
+    grad = mlp.prediction_gradient(X)
+    grad_means = grad.mean(axis=0)
+    assert grad.shape == X.shape
+    # Check that only the informative feature has a large gradient.
+    assert np.abs(grad_means[0]) > 0.5
+    for m in grad_means[1:]:
+        assert np.abs(m) < 0.1
+
```

Comment:
It looks like `make_regression` can handle multiclass problems (with `n_targets`) - maybe add the multiclass test from the MLPClassifier here too?

Reply:
MLPRegressor doesn't do multilabel yet, unfortunately (#85)